server/controllers/events/events_controller.go

```diff
@@ -377,7 +382,8 @@ func (e *VCSEventsController) handleBitbucketServerPullRequestEvent(w http.Respo
 	}
 	pullEventType := e.Parser.GetBitbucketServerPullEventType(eventType)
 	e.Logger.Info("identified event as type %q", pullEventType.String())
-	resp := e.handlePullRequestEvent(e.Logger, baseRepo, headRepo, pull, user, pullEventType)
+	eventTimestamp := time.Now()
```

Comment:
why are we using `time.Now()` is it because we don't have anything in the event itself and we're planning on generating the timestamp in the proxy itself?

Reply:
Thanks for pointing out, meant to go back and add GH timetstamps, but leave `time.Now()` for other VCS clients for now. In those cases, the timestamp is being generated in proxy.